pkg/controller/amazoncloudintegration/amazoncloudintegration_controller.go

```diff
@@ -194,18 +220,18 @@ func (r *ReconcileAmazonCloudIntegration) Reconcile(ctx context.Context, request
 	}
 	component, err := render.AmazonCloudIntegration(amazonCloudIntegrationCfg)
 	if err != nil {
-		r.SetDegraded("Error rendering AmazonCloudIntegration", err, reqLogger)
+		r.SetDegraded(operatorv1.ResourceRenderingError, "Error rendering AmazonCloudIntegration", err, reqLogger)
 		return reconcile.Result{}, err
 	}
 
 	err = imageset.ApplyImageSet(ctx, r.client, variant, component)
 	if err != nil {
-		r.SetDegraded("Error with images from ImageSet", err, reqLogger)
+		r.SetDegraded(operatorv1.ResourceUpdateError, "Error with images from ImageSet", err, reqLogger)
```

Comment:
If you step into the code, you will see that an error stems from either:
- Reading
- Validating

It seems that there is not a fitting error right now. Perhaps we could generalize it to "ImageSetError"?

Reply:
Done